src/main/java/com/uid2/admin/vertx/AdminVerticle.java

```diff
@@ -57,19 +59,20 @@ public void start(Promise<Void> startPromise) {
 
     private Router createRoutesSetup() {
         final Router router = Router.router(vertx);
-        router.route().handler(SessionHandler.create(LocalSessionStore.create(vertx)));
+        router.route().handler(SessionHandler.create(LocalSessionStore.create(vertx)).setSessionTimeout(32400000)); // 9 hr session timeout
         final AuthenticationHandler oktaHandler = this.authProvider.createAuthHandler(vertx, router.route("/oauth2-callback"));
+
         router.route().handler(BodyHandler.create());
         router.route().handler(StaticHandler.create("webroot"));
 
         router.route("/login").handler(oktaHandler);
-        router.route("/adm/*").handler(oktaHandler); 
+        router.route("/adm/*").handler(oktaHandler);
```

Comment:
do we still have routes with /adm/* ? Or do they go away when you delete AdminKeyService.java ?

Reply:
/adm/* matches against UI interactions by webroot/adm/*.html